flash/vision/classification/data.py

```diff
@@ -446,51 +477,101 @@ def from_folder(
             folder/cat_asd932_.png
 
         Args:
-            folder: Path to the data folder.
-            transform: Image transform to apply to the data.
-            loader: A function to load an image given its path.
-            batch_size: Batch size for data loading.
+            train_filepaths: String or sequence of file paths for training dataset. Defaults to ``None``.
+            train_labels: Sequence of labels for training dataset. Defaults to ``None``.
+            valid_split: If not None, generates val split from train dataloader using this value.
+            valid_filepaths: String or sequence of file paths for validation dataset. Defaults to ``None``.
+            valid_labels: Sequence of labels for validation dataset. Defaults to ``None``.
+            test_filepaths: String or sequence of file paths for test dataset. Defaults to ``None``.
+            test_labels: Sequence of labels for test dataset. Defaults to ``None``.
+            train_transform: Transforms for training dataset. Defaults to ``default``, which loads imagenet transforms.
+            valid_transform: Transforms for validation and testing dataset.
+                Defaults to ``default``, which loads imagenet transforms.
+            batch_size: The batchsize to use for parallel loading. Defaults to ``64``.
```

Comment:
```suggestion
            train_filepaths: String or sequence of file paths for training dataset.
            train_labels: Sequence of labels for training dataset.
            valid_split: If not None, generates val split from train dataloader using this value.
            valid_filepaths: String or sequence of file paths for validation dataset.
            valid_labels: Sequence of labels for validation dataset.
            test_filepaths: String or sequence of file paths for test dataset.
            test_labels: Sequence of labels for test dataset.
            train_transform: Transforms for training dataset. Defaults to ``default``, which loads imagenet transforms.
            valid_transform: Transforms for validation and testing dataset.
                Defaults to ``default``, which loads imagenet transforms.
            batch_size: The batchsize to use for parallel loading.
```

Reply:
I think the defaults in the args is fine.